runtime/tests/interop/common.qinc

```diff
@@ -2,7 +2,7 @@
 
     class EntrypointSetup {
         EntrypointSetup() {
-            logging.config().setLevel("debug").setAppender(logging.stdout()).configure();
+            logging.defaultConfig().setLevel("debug").setAppender(logging.stdout()).configure();
```

Comment:
The config() method is not actually an accessor for a default, but a factory method for the Config interface. I'm assuming the change was necessary due to the conflict between config the method and Config the class. Bozzo suggested maybe renaming the Config class to Configurator instead, or possibly changing config() to makeConfig().